### DISCUSSION TITLE
Integrate slope/gradient/terrain information.

### DISCUSSION THREAD

**Discussion by xanthking** [skarkkai/touch-mapper#50] [Ideas]
It would be very useful for finding the most efficient route with the least hills or for a user to have a heads up of how much uphill walking they may have to do to reach a destination. If this can include buildings and streets, it would be very empowering.

**Comment by skarkkai** [comment]
Does this idea come from a concrete need of some visually impaired people?

**Comment by majdal** [comment]
I came here to ask for this too. I don't come to this from the visually impaired perspective, but from an artist perspective. It would make the pieces even more appealing.
